.chromatic/preview-head.html

```diff
@@ -1,24 +1,56 @@
 <link rel="preload" href="https://use.typekit.net/uma8ayv.css" as="style" />
+<link rel="preload" href="https://use.typekit.net/gaa0vrf.css" as="style" />
 
 <link rel="preload" href="https://use.typekit.net/af/cb695f/000000000000000000017701/27/l?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=n4&v=3" as="font" crossorigin="anonymous" />
-<link rel="preload" href="https://use.typekit.net/af/cb695f/000000000000000000017701/27/d?primer=7cdcb44be4a7db8877ffa5c0007b8dd865b3bbc383831fe2ea177f62257a9191&fvd=n4&v=3" as="font" crossorigin="anonymous" />
```

Comment:
interesting, where'd this come from?

Reply:
In the typekit files we are preloading it includes all three versions of a font. This is done so if it can't load one it falls back to the next and tries that and stops when one loads. In the preloading previously, we were preloading each font which would download all versions of the font, even though we only need one.

In the typekit files they define what type (woff2, woff, and opentype) each url is, this is how I figured out that 'l' was woff2, 'd' was woff, and 'a' is opentype. To determine which version to use I read about the differences online.